integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java

```diff
@@ -266,9 +284,15 @@ private static String[] optimizeAndTransformFuseMountOptions(AlluxioConfiguratio
     // See https://github.com/libfuse/libfuse/blob/fuse_2_9_3/ChangeLog#L655-L659,
     // and https://github.com/torvalds/linux/commit/78bb6cb9a890d3d50ca3b02fce9223d3e734ab9b.
     // Libfuse3 dropped this option because it's default. Include it doesn't error out.
-    options.add("-obig_writes");
+    if (AlluxioFuseUtils.getVersionPreference(conf) == VersionPreference.VERSION_2) {
```

Comment:
the loaded libfuse version may not be the version specified in the config. what do you think?

Reply:
This will need to depend on your PR? the one that has test failure waiting for me to fix, currently the libfuse loaded version i guess is always 2?

Reply:
Ah right.